contracts/SignerBondsEscrow.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity <0.9.0;
+
+import "./RiskManagerV1.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+/// @title SignerBondsEscrow
+/// @notice Process incoming signer bonds by putting them into escrow the
+///         governance can withdraw from.
```

Comment:
We need to be very precise with documentation so that it is clear what is the purpose of this strategy and that it is all about *swapping manually* and not withdrawing. I would suggest something like:

ETH purchased by the risk manager from tBTC signer bonds needs to be swapped and deposited back to the coverage pool as collateral. In the case it can not be done automatically, the governance has the power to ask the risk manager to deposit ETH from purchased signer bonds into an escrow the governance can later withdraw from and do the swap manually. SignerBondsEscrow is a simple escrow implementation allowing the risk manager to store purchased ETH signer bonds so that governance can later swap them manually and deposit as coverage pool collateral.

Reply:
See https://github.com/keep-network/coverage-pools/pull/39/commits/fdcf4dddddb007e10c1f6540980b81996962ed1d